ghost/members-api/test/unit/lib/repositories/member.test.js

```diff
@@ -404,6 +458,57 @@ describe('MemberRepository', function () {
             }).should.be.rejectedWith(`Cannot subscribe to invalid newsletter ${INVALID_NEWSLETTER_ID}`);
         });
 
+        it('does not allow a member to be subscribed if any newsletter id from the list is invalid', async function () {
```

Comment:
I don't think we need this test, the same behaviour is already covered in `does not allow a member to be subscribed to an invalid newsletter`